src/components/Sum.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <p>
+    {`Sum of ${a || 0} and ${b || 0} is ${(a || 0) + (b || 0)}`}
```

Comment:
More simple to set default value in destructuring, like `a = 0, ...` or with 'Sum.defaultProps = {}'